fabtests/benchmarks/benchmark_shared.c

```diff
@@ -460,17 +486,25 @@ int bandwidth_rma(enum ft_rma_opcodes rma_op, struct fi_rma_iov *remote)
 							tx_buf + offset,
 							opts.transfer_size,
 							remote);
+				} else if (opts.use_fi_more) {
+					flags = set_fi_more_flag(i, j, flags);
+					ret = ft_post_rma(FT_RMA_WRITEMSG, 
+							tx_buf + offset,
+							opts.transfer_size, 
+							remote, &tx_ctx_arr[j].context,
+							flags);
 				} else {
 					ret = ft_post_rma(FT_RMA_WRITEDATA,
 							tx_buf + offset,
 							opts.transfer_size,
-							remote,	&tx_ctx_arr[j].context);
+							remote,	&tx_ctx_arr[j].context,
+							flags);
```

Comment:
Hmmm.. do you expect a non-zero flags to be used here? Similar to other `ft_post_rma` that is used for non-fi-more.

Reply:
I should add `flags |= FI_REMOTE_CQ_DATA` for writedata with fi_more.
Do you mean we should just pass 0 for the others?

Reply:
My asking is that before your change, `flag` is not an argument of `ft_post_rma`, if we only extend `ft_post_rma` to use flags for `FI_MORE` usage, should we keep flags as 0 for non-fi-more case?

Reply:
Yeah that makes sense